### ISSUE TITLE
3.1 OS metrics - doesn't show correct disk in pie graphs

### ISSUE THREAD

**Issue by slivne** [scylladb/scylla-monitoring#768]
Using branch-3.0

```
[centos@ip-10-0-0-186 ~]$ df
Filesystem     1K-blocks     Used Available Use% Mounted on
devtmpfs         7784668        0   7784668   0% /dev
tmpfs            7812204        0   7812204   0% /dev/shm
tmpfs            7812204    16820   7795384   1% /run
tmpfs            7812204        0   7812204   0% /sys/fs/cgroup
/dev/xvda1      31445996  7233508  24212488  24% /
/dev/nvme0n1   463640688 34331196 429309492   8% /var/lib/scylla
tmpfs            1562444        0   1562444   0% /run/user/1000
```

![3 1_disk_incorrect](https://user-images.githubusercontent.com/3465480/67625061-bf721c00-f841-11e9-82e4-f3992161506a.png)

**Comment by amnonh** [comment]
It's probably fixed by c155342a9f0413fe3b13baa854c0f0f6b869a124 we can either backport it, or branch 3.1

**Comment by slivne** [comment]
branch-3.0 115b7b0ef7d295d628b14f96ca1e9913f0b28030

**Comment by amnonh** [comment]
closed in 3.0.2
